SQL/0000-00-02-Menus.sql

```diff
@@ -71,8 +71,6 @@ INSERT INTO LorisMenuPermissions (MenuID, PermID)
 
 -- Access Profile
 INSERT INTO LorisMenuPermissions (MenuID, PermID)
-    SELECT m.ID, p.PermID FROM permissions p CROSS JOIN LorisMenu m WHERE p.code='data_entry' AND m.Label='Access Profile';
```

Comment:
This will make the Menu item visible to everyone.
I think you should add a row for 'access_all_profiles'